tests/data/defs-aliases-chargeconj.dec

```diff
@@ -276,3 +276,13 @@ SetLineshapePW D_10 D*0 pi0 2
 SetLineshapePW D_10 D*+ pi- 2
 SetLineshapePW anti-D_10 anti-D*0 pi0 2
 SetLineshapePW anti-D_10 D*- pi+ 2
+
+
+# Model aliases used in Belle II
+ModelAlias SLBKPOLE_DtoKlnu SLBKPOLE 1.0 0.303 1.0 2.112;
```

Comment:
Would a line such as `ModelAlias SLBKPOLE_DtoKlnu SLBKPOLE 1.0 0.303 1.0 2.112 ;` work? Then add a test?

Reply:
Yes, this works. I've added a sub-test in the relevant function.